src/pages/Landing/index.tsx

```diff
@@ -303,9 +334,36 @@ export default function Landing() {
   const cardsRef = useRef<HTMLDivElement>(null)
   const selectedWallet = useAppSelector((state) => state.user.selectedWallet)
   const shouldDisableNFTRoutes = useDisableNFTRoutes()
-  const cards = useMemo(
-    () => MAIN_CARDS.filter((card) => !(shouldDisableNFTRoutes && card.to.startsWith('/nft'))),
-    [shouldDisableNFTRoutes]
+  const originCountry = useAppSelector((state: AppState) => state.user.originCountry)
+  const renderUkSpecificText = Boolean(originCountry) && originCountry === 'GB'
```

Comment:
Just confirming - will origin country be `GB` or `UK`?

Reply:
GB is the two character ISO code for UK